Essentials/src/com/earth2me/essentials/EssentialsTimer.java

```diff
@@ -89,6 +89,7 @@ public void run() {
                 continue;
             }
             user.checkMuteTimeout(currentTime);
+            user.checkIgnoreTimeouts(currentTime);
```

Comment:
As @md678685 said, if you could share timings before and after this change that would help us determine if this is safe to have sync.

Reply:
I don't have any timings available, but I think this would have scaled badly to larger servers where more people would use the ignore command. I think the different method suggested by @pop4959 is the solution for this, so have moved my processing code to that method for now.